src/helpers/general.helpers.ts

```diff
@@ -0,0 +1,29 @@
+export const fileExists = async (file: string): Promise<boolean> => {
+  try {
+    await Deno.stat(file);
+    return true;
+  } catch (e) {
+    if (e instanceof Deno.errors.NotFound) {
+      return false;
+    }
+    throw e;
+  }
+};
+
+export async function runCli(params: string) {
+  const props = params.split(" ");
+
+  const process = Deno.run({
+    cmd: props,
+    stdout: "piped",
+    stderr: "piped",
+  });
+
+  const { code } = await process.status();
+  if (code === 0) {
+    return new TextDecoder().decode(await process.output()).trim();
+  } else {
+    const error = new TextDecoder().decode(await process.stderrOutput());
+    console.error(error);
+  }
+}
```

Comment:
issue(blocking): this helper isn't reusable and can break things.
Imagine you pass something like `git commit -m "This message also has spaces"` - this helper will split the string incorrectly and break the application.

We could keep `Deno.run` as is in the caller function and create a helper that would only decode and handle errors like the following example below, but I would suggest getting rid of this helper and handling this in place.
My reasoning for not using a helper for this is that not all errors should be thrown, some should be silenced, and others might actually require not only console logging but also exiting the application completely. It's not worth writing a generic helper at this point.

```suggestion
export async function tryDecode(process: Process) {
  const { code } = await process.status();
  
  if (code !== 0) {
    const error = new TextDecoder().decode(await process.stderrOutput());
    console.error(
      `[%cERROR%c] Process failed with an error: ${error}`,
      "color: red",
      "color: inherit",
    );
  }

  return new TextDecoder().decode(await process.output()).
}
```